VERSION

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 6.0.14-dev
+=======
+7.0.0
+>>>>>>> 7.0.0-tmp
```

Comment:
I think those lines show what came from the 7.0.0 branch and what came from master (HEAD).  I don't think they should be checked in.